src/lib/ng-v5/init/analyse-sources.transform.ts

```diff
@@ -42,13 +42,23 @@ function analyseEntryPoint(graph: BuildGraph, entryPoint: EntryPointNode, entryP
     readResource: () => ''
   };
 
+  // we cannot re-use an ng program without emitting
```

Comment:
I don't see the value of this comment here.

Reply:
Because strictly speaking we should reuse a program here 😊

Reply:
Ah, I see now. Perhaps the comment can be made more clear then:

```
A new program is created here, as program reuse is only possible when emitting which is not done for entry-point analysis.
```